domains/*.business.is-local.org.json

```diff
@@ -10,7 +10,8 @@
     },
 
     "record": {
-        "A": ["13.51.77.148"]
+        "A": ["13.51.77.148"],
+        "NS": ["burt.ns.cloudflare.com", "desi.ns.cloudflare.com"]
```

Comment:
If you want NS records, you must provide the reason as to **why you need** NS records. You also must remove the A record.

Reply:
Apologies, I had no idea, I would need NS records to be able to create subdomains for different projects on top of the original subdomain. I thought wildcard subdomains were meant for that purpose, that is why I applied for one. 

I am also currently running my project on an AWS free tier EC2 and last time the instance glitched out, I had to shut it down and boot it back up which then changed my instance IP, so I thought it would make the whole process easier if I linked the subdomain to Cloudflare DNS, I wouldn't need to bug someone into reviewing a pull request for an A record IP change.

Should I delete this pull request and make a new one without the A record? The bot also didn't ask about CLA, so I didn't know I need to agree to the terms again.

Reply:
You can just make the changes here and you should be good. Your NS record reasoning seems good, as usual, do not abuse these records.

Reply:
Of course, and thank you, I will make the necessary changes right away.